bugbug/repository.py

```diff
@@ -365,7 +365,10 @@ def get_touched(functions, lines):
 
 
 def get_metrics(commit, metrics_space):
-    if metrics_space["kind"] == "function" and metrics_space["name"]:
+    name = metrics_space["name"]
+    error = metrics_space["kind"] in {"unit", "function"} and name == ""
```

Comment:
unit should always have a name (the file name in fact)

Reply:
Yep, so I'm going to return an error if the kind is either function or unit and the name is missing